iotp-adaptor/src/main/webapp/WEB-INF/web.xml

```diff
@@ -12,17 +12,17 @@
   <context-param>
     <description>Scheme used for URI when registering ServiceProvider.  Can be overridden by System property of the same name.</description>
     <param-name>com.ibm.oslc.adaptor.iotp.servlet.scheme</param-name>
-    <param-value>https</param-value>
+    <param-value>http</param-value>
   </context-param>
   <context-param>
     <description>Base URI for the adaptor.</description>
     <param-name>com.ibm.oslc.adaptor.iotp.servlet.baseurl</param-name>
-    <param-value>https://rlia4iot.raleigh.ibm.com:9443</param-value>
+    <param-value>http://192.168.1.91:8080</param-value>
```

Comment:
I think localhost or 127.0.0.1 would be better.

Reply:
Yes, sure, I forgot to reset those values. Thanks!